parsl/launchers/launchers.py

```diff
@@ -8,16 +8,20 @@
 class SimpleLauncher(Launcher):
     """ Does no wrapping. Just returns the command as-is
     """
-    def __init_(self, debug: bool = True) -> None:
+    def __init__(self, debug: bool = True) -> None:
         super().__init__(debug=debug)
 
-    def __call__(self, command: str, tasks_per_node: int, nodes_per_block: int) -> str:
+    def __call__(self, command: str, tasks_per_node: int, nodes_per_block: int, permit_multiple_nodes: bool = False) -> str:
         """
         Args:
         - command (string): The command string to be launched
-        - task_block (string) : bash evaluated string.
-
+        - tasks_per_node (int): Number of tasks to launch per node
+        - nodes_per_block (int): Number of nodes per block
+        - permit_multiple_nodes (bool): Whether to allow multiple nodes per block
```

Comment:
In the discussion of issue #3161 , @Andrew-S-Rosen and I concluded that we probably didn't need this parameter if issuing a warning (instead of raising an exception) so it would probably be good to remove this extra complexity.